src/renderer/helpers/getTypeAccentColors.ts

```diff
@@ -0,0 +1,28 @@
+import { evaluate } from '../../common/types/evaluate';
+import { NamedExpression } from '../../common/types/expression';
+import { intersect } from '../../common/types/intersection';
+import { TypeDefinitions } from '../../common/types/typedef';
+import { NumberType, StringType, Type } from '../../common/types/types';
+
+const defaultColor = () => '#718096';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const colorList = (typeDefinitions: TypeDefinitions, isDarkMode = true) => [
```

Comment:
Please don't ignore the unused var.

Reply:
I'm going to remove it from here but keep it (and ignore it) in the other function, as It could potentially be useful in the future and I don't want to remove all the other code that passes the dark mode value in.